### ISSUE TITLE
Tutorials for scIB

### ISSUE THREAD

**Issue by mbuttner** [theislab/scib#182]
Hi there,

I think that having tutorials for the different metrics would be very cool.

**Comment by LuckyMD** [comment]
Yes :). Maybe we can make this a milestone and do project management via github here.
